### ISSUE TITLE
Building docs repeatedly takes a long time each time

### ISSUE THREAD

**Issue by maxalbert** [Flowminder/FlowKit#772]
Building the docs takes a reasonable amount of time because it needs to spin up the `flowb_synthetic_data` container, which takes 4-5 minutes.

One would expect that once
```
cd docs/
pipenv run build
```
has been run (which takes a few minutes), a subsequent
```
pipenv run build/serve
```
would be much quicker because it should have already generated the data. However, the second command takes the same amount of time because it needs to re-generate the data. This is highly inconvenient because it makes iterating over docs builds very slow.

**Comment by maxalbert** [comment]
While this is an inconvenience rather than a bug, it hints at an issue that has come up repeatedly in the past - namely, whether the various flowdb containers should persist their data in external volumes or not. I have opened issue #773 to discuss this further and keep it separate from this particular issue about the docs.

**Comment by greenape** [comment]
For the iterating - serve will rebuild any changed pages and trigger a reload in the browser (also works for changes to docstrings). Won't help if you're changing the structure of the docs tho.

**Comment by maxalbert** [comment]
Yeah, using `serve` should mitigate this issue. However, whenever I tried it in the past it didn't always update things the way I expected it to (even if there were no structural changes AFAICR) so I have tended to avoid it. But I'll give it another go next time I work with the docs. 👌
